.github/workflows/codeql-analysis.yml

```diff
@@ -26,7 +26,7 @@ permissions:
 jobs:
   CodeQL-Build:
     runs-on: ubuntu-latest
-    if: github.event.pull_request.draft != true
+    if: github.event.pull_request.draft = false
```

Comment:
I'm commenting here to prevent us accidentally merging this PR. Looks like I'll need to mark this "ready to go" to test my changes....

Reply:
Is it always set to `false` if false?

Reply:
I don't understand what you mean... Sorry! 🙂

I think the code is meant to skip certain tests if the PR is marked as a _draft_?

Reply:
I mean it might be set to `true` when it's draft, but undefined when it's not - needs checking

Reply:
doesn't it need to be `==` double equal?